guides/common/modules/proc_booting-virtual-machines.adoc

```diff
@@ -48,19 +48,14 @@ If you want to change the default values in the template, clone the template and
 . Set *PXE Loader* to *iPXE Embedded*.
 . Select the *Templates* tab.
 . From the *iPXE template* list, select *Review* to verify that the *Kickstart default iPXE* template is the correct template.
-. Configure the `dhcpd.conf` file as follows:
-+
+
+On {ProjectServer}, run:
```

Comment:
```suggestion
. On {ProjectServer}, run:
+
```

Reply:
This is merging `On ProjectServer, run` with the 18th point of the procedure. Also, `+` is visible if I don't move `On ProjectServer, run` to the 51st line. `+` seems not required to me.

Reply:
Previous sentence talks about a separate action in the Project Web UI so I think the command should be its own step. @ekohl ?

Reply:
Okay. Does this PR looks good?